doc/admin-guide/files/sni.yaml.en.rst

```diff
@@ -61,6 +61,12 @@ ip_allow                  Inbound   Specify a list of client IP address, subnets
                                     the connection. This list is comma separated. IPv4 and IPv6 addresses can be specified.
                                     Here is an example list: 192.168.1.0/24,192.168.10.1-4. This would allow connections
                                     from clients in the 19.168.1.0 network or in the range from 192.168.10.1 to 192.168.1.4.
+                                    Alternatively, the path to a file containing
+                                    the list of comma-separated IP addresses can
+                                    be specified in the form of
+                                    ``@path_to_file``. If a given file path does
```

Comment:
> In addition to ' and " there are a number of characters that are special (or reserved) and cannot be used as the first character of an unquoted scalar: [] {} > | * & ! % # ` @ ,.

https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

We may want to be more specific about this (maybe with a sample) since the default file uses unquoted scalars and just replacing unquoted `127.0.0.1` with unquoted `@/x/y/z` can mean something else.

Reply:
Good point. I updated the documentation to make that clear and add an example. See if you like the text and format.

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/19499960/223867528-7a1b8abb-5cbe-4271-a06f-6378107ac293.png">

Reply:
Looks good. Thanks.